front/src/applications/stdcmV2/components/StdcmConfig.tsx

```diff
@@ -71,24 +94,15 @@ const StdcmConfig = ({
     <div className="stdcm-v2__body">
       <div className="stdcm-v2-simulation-settings">
         <div className="stdcm-v2-consist-container">
-          <StdcmConsist
-            disabled={isPending}
-            setCurrentSimulationInputs={setCurrentSimulationInputs}
-          />
+          <StdcmConsist {...inputsProps} />
         </div>
         <div className="stdcm-v2__separator" />
         <div className="stdcm-v2-simulation-itinerary">
           {/* //TODO: use them when we implement this feature #403 */}
           {/* <StdcmDefaultCard text="Indiquer le sillon antérieur" Icon={<ArrowUp size="lg" />} /> */}
-          <StdcmOrigin
-            disabled={isPending}
-            setCurrentSimulationInputs={setCurrentSimulationInputs}
-          />
-          <StdcmVias disabled={isPending} setCurrentSimulationInputs={setCurrentSimulationInputs} />
-          <StdcmDestination
-            disabled={isPending}
-            setCurrentSimulationInputs={setCurrentSimulationInputs}
-          />
+          <StdcmOrigin {...inputsProps} />
+          <StdcmVias {...inputsProps} />
```

Comment:
It looks like when I'm changing the stop duration of a via and leaving the focus, the pathfinding is relaunched but it shouldn't be the case. The stop duration is only necessary for the stdcm endpoint, not the pathfinding one.

Reply:
Yes, that's because changing the stop duration triggers the `updateViaStopTimeV2` action, which looks like this:
```typescript
updateViaStopTimeV2(
      state: Draft<S>,
      action: PayloadAction<{ via: PathStep; duration: string }>
    ) {
      const {
        payload: { via, duration },
      } = action;
      state.pathSteps = state.pathSteps.map((pathStep) => {
        if (pathStep && pathStep.id === via.id) {
          return { ...pathStep, stopFor: duration };
        }
        return pathStep;
      });
    },
```
We can see that we modify the pathStep, so the pathfinding is relaunched.